skip-to/skip-to.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
I think this folder should be in the [`mu-plugins` folder](https://github.com/WordPress/wporg-mu-plugins/tree/trunk/mu-plugins/), if I'm understanding the repo structure correctly

Reply:
Good catch, yeah, `mu-plugins` is what's synced to `pub-sync` on dotorg.

Reply:
Ah... Yeah good catch!

The structure of this repo doesn't allow this repo to be developed on sandboxes, so I screwed up in the copy-to-git process.

Reply:
yeah, that is annoying 🙁 

related: https://github.com/WordPress/wporg-mu-plugins/issues/3